### ISSUE TITLE
Custom arrows and bolts

### ISSUE THREAD

**Issue by h4ppyboy** [SlimeKnights/TinkersConstruct#5154]
### Minecraft Version

1.18.2

### Forge Version

any

### Mantle Version

any

### Tinkers' Construct Version

1.18.2-3.6.4.113

### Problem description

No custom arrows like in 1.12.2

### Suggested solution

Add custom arrows back, a lot of players liked the previous system with custom made arrows with modifiers and traits you like

### Alternatives considered

no alternatives, this need only adding previous custom arrows

### Searched for existing enhancement?

Used the search bar, Checked the FAQ

**Comment by ShinyPorygon** [comment]
This system is not returning. If you want more arrows you can use a mod such as Archers Paradox. Or you can go play 1.12 which has said system. 

You also claimed to have Checked the FAQ clearly you missed the section telling you exactly why they are not returning.
https://github.com/SlimeKnights/TinkersConstruct/wiki/New-in-Tinkers%27-Construct-3#removed-tools

**Comment by h4ppyboy** [comment]
Maybe at least an addon of good old arrows for 1.18?

**Comment by ShinyPorygon** [comment]
You are more than welcome to make an add-on. Nothing will come from Tinkers. If you make an add-on you will also have to rework bows and remove most of their modifiers that would go on 1.12 style arrows. If you would like to discuss any of this further feel free to join the discord.  https://discord.gg/njGrvuh

**Comment by h4ppyboy** [comment]
k, but i don't understand what's wrong with old design of arrows ? and why vanilla one's are better. I'd like to understand why, in some simple words

**Comment by KnightMiner** [comment]
I have talked about why arrows are not returning in #5144, along with on the [What's New](https://github.com/SlimeKnights/TinkersConstruct/wiki/New-in-Tinkers%27-Construct-3#removed-tools) page on the wiki. Said page is conveniently linked on the issue template selection screen.

Short version: using durability as ammo was dumb and just lead to a lot of confusion and exploits. Modifying stacks of items is way too expensive for a disposable item. Instead, just think of vanilla arrows as having 1 modifier slot that you can fill with potions, glowstone, or various things from arrow mods.

**Comment by h4ppyboy** [comment]
nah, durability as ammo was a genial idea with bad interpretation, it was comfortable and easy, i don't think someone was confused on this. It was really awkward to see 4120 arrows, but it wasn't a good cause to remove arrows

**Comment by KnightMiner** [comment]
Just because you understood it does not mean that no one was confused. I am the developer of the mod. I talked to the people who were confused. They were confused.

And yes, one problem was not enough to remove them. I did not just say one problem, I said multiple. I also said it led to exploits, the many modifiers we have to boost durability were not designed to boost ammo, it made all ammo trivial and made vanilla arrows useless.

Combine that with the fact there were no clear rules about whether a modifier goes on the bow or arrow, the fact that it was twice as much effort to design good bows than good tools, and the fact that there were so few good modifiers for either so you rarely had to make interesting choices, and you got a bad design all around.

Let me also remind you, they were not removed. They were scrapped as not worth bringing back. I analyzed multipart arrows and decided they did not add enough to be worth making the design work nor worth the trouble coding. Bows are a lot more interesting and a lot more compatible with other mods when they just use vanilla arrows, it means all the arrow modifiers for things like looting and boosting arrow damage just end up on the bow instead, giving you plenty of good modifier choices.
